### PR TITLE
add "Getting support" page covering EESSI support portal

### PR DESCRIPTION
**Pull request by laraPPr**
*This is a first draft for the new page related to support portal of EESSI*

### CODE REVIEW

**Comment by casparvl** [comment]
This document now treats what is in scope, and what is out of scope. Should we also say something about the level of support that can be expected (I'd say "best effort" or something)? Similarly, should we mention that we reserve the right to prioritize in any way we see fit? (I'm guessing in practice this will be: MultiXscale related issues come first, as they should, since that's how this whole support thing is funded)

File: docs/support.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+You can find lists on the information that we would like to receive in the ReadMe of the GitLab repository.
+
+## What do we support
+
+### Getting access the EESSI software stack
+If you have trouble connecting to the software stack. 
+Such as trouble related the [cernVM-FS filesystem layer](filesystem_layer.md) or the [compat layers](compatibility_layer.md). 
+Please contact us. 
+
+### software requests
```
Comment (ocaisa) [inline docs/support.md:25]:
```suggestion
### Software requests
```

File: docs/support.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+We are open to software requests for applications that aren't available. You can try it your self by looking through the [documentation on adding software](software_layer/adding_software.md). 
+Or you can just send in a request. 
+Please try to provide as much information on the software as possible.
```
Comment (ocaisa) [inline docs/support.md:28]:
```suggestion
Please try to provide as much information on the software as possible, and be aware that we can only provide software that has an appropriate open source license.
```

File: docs/support.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Test-suite
+If you are using the test-suite and run into any trouble using it you can also sent them in via our GitLab support portal or email.
```
Comment (ocaisa) [inline docs/support.md:31]:
```suggestion
If you are using the test-suite and run into any trouble using it you can also report your issues via our GitLab support portal or email.
```

Comment (ocaisa) [reply docs/support.md:31]: We should probably add the link to the GitLab support portal here

File: docs/support.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Using the software
+Please report any problems that you encounter when using software in the EESSI stack that are specifically related to the installation within EESSI.
```
Comment (ocaisa) [inline docs/support.md:34]: This probably needs some more level of detail, like example of what we would support, and something we would not.

File: docs/support.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+### Test-suite
+If you are using the test-suite and run into any trouble using it you can also sent them in via our GitLab support portal or email.
+
+### Using the software
+Please report any problems that you encounter when using software in the EESSI stack that are specifically related to the installation within EESSI. 
+
+## What do we not support
+
+We ask the users of EESSI to refrain from sending in issues that are specifically related to helping them use certain software that EESSI provides.
```
Comment (ocaisa) [inline docs/support.md:38]:
```suggestion
We ask the users of EESSI to refrain from sending in issues that are specifically related to
helping them use certain software that EESSI provides. EESSI does not the capacity to have expertise
on all the software that it provides and you are more likely to get better support from the software
project itself.
```

**Review by boegel — APPROVED** [review]

**Pull request merged**